src/components/v5/common/Table/Table.tsx

```diff
@@ -0,0 +1,142 @@
+    columns: burgerColumn ? [...columns, ...burgerColumn] : columns,
+    getCoreRowModel: getCoreRowModel(),
+  });
+
+  const HeaderMobileTable = ({ headerGroups, lastColumn }) =>
+    headerGroups.map((headerGroup) => (
+      <div
+        key={headerGroup.id}
+        className={`flex flex-col bg-gray-50 rounded-t-lg border-gray-200 
+      capitalize text-gray-600 gap-[1.3rem] py-2 text-sm w-[7.5rem] border-r`}
+      >
+        {headerGroup.headers.map((header) => (
+          <div
+            key={header.id}
+            className={clsx(
+              'text-left text-sm text-gray-600 font-normal rounded-t-lg px-4',
+              { hidden: lastColumn.column.id === header.id },
+            )}
+          >
+            {header.isPlaceholder
+              ? null
+              : flexRender(header.column.columnDef.header, header.getContext())}
+          </div>
+        ))}
+      </div>
+    ));
+
+  return (
+    <div className={className}>
+      {tableTitle && <h5 className="text-2 mb-3">{tableTitle}</h5>}
+      <div className="border border-gray-200 rounded-lg">
+        <table className="w-full">
+          <thead
+            className={clsx(
+              'h-[2.1875rem] bg-gray-50 rounded-t-lg border-b border-gray-200',
+              { hidden: isMobile },
+            )}
+          >
+            {table.getHeaderGroups().map((headerGroup) => (
+              <tr key={headerGroup.id}>
+                {headerGroup.headers.map((header) => (
+                  <th
+                    key={header.id}
+                    className="text-left text-sm text-gray-600 font-normal rounded-t-lg px-4"
+                  >
+                    {header.isPlaceholder
+                      ? null
+                      : flexRender(
+                          header.column.columnDef.header,
+                          header.getContext(),
+                        )}
+                  </th>
+                ))}
+              </tr>
+            ))}
+          </thead>
+          <tbody className="w-full">
+            {table.getRowModel().rows.map((row) => {
+              const lastRow =
+                table.getRowModel().rows[table.getRowModel().rows.length - 1];
+              const lastColumn =
+                row.getVisibleCells()[row.getVisibleCells().length - 1];
+
+              return (
+                <tr
+                  key={row.id}
+                  className={clsx('relative', {
+                    'border-b border-b-gray-200': lastRow.id !== row.id,
+                    'w-full inline-flex': isMobile,
+                    'h-[3.125rem] border-none': !isMobile,
+                  })}
+                  onClick={() => setSelectedRowId(row.id)}
+                >
+                  {isMobile && (
+                    <HeaderMobileTable
+                      headerGroups={table.getHeaderGroups()}
+                      lastColumn={lastColumn}
+                    />
+                  )}
+
+                  <div
```

Comment:
You can't use a div like this inside a table.

Reply:
I know but there is no other way to display this table horizontally likeon the design. This library not supports mobile version.